debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -64,6 +70,7 @@ public class RedisStreamChangeConsumer extends BaseChangeConsumer
     private static final String PROP_WAIT_TIMEOUT = PROP_PREFIX + "wait.timeout.ms";
     private static final String PROP_WAIT_RETRY_ENABLED = PROP_PREFIX + "wait.retry.enabled";
     private static final String PROP_WAIT_RETRY_DELAY = PROP_PREFIX + "wait.retry.delay.ms";
+    private static final String PROP_MEMORY_THRESHOLD = PROP_PREFIX + "memory.threshold";
```

Comment:
I think that `memory.threshold.percentage` is better as the units are already there like in other properties' suffixes (.ms, etc.)